### PR TITLE
chore: #1034 enable documentation autorefresh for jkit and gradle doc

### LINKED ISSUE #1034: Enable documentation auto refresh while editing adoc files.
### Description
When editing the JKube kubernetes-maven-plugin documentation, it's painful as the writer has to rerun the command mvn -Phtml package each time and it may takes some time to perform the whole generation.

For the kubernetes maven plugin: 
- [x] https://github.com/eclipse/jkube/pull/983
- [x] https://github.com/eclipse/jkube/pull/996 (requiring CQ to be created)

For jube-kit
- [x] maybe add the same profile for https://github.com/eclipse/jkube/tree/master/jkube-kit/doc : https://github.com/eclipse/jkube/pull/1122

### PR DESCRIPTION
**Pull request by sunix**
## Description
Fix #1034
This is finalising https://github.com/eclipse/jkube/issues/1034
1. extract common properties to the jkube parent pom.xml
2. added doc-watch profile for jkit and gradle doc

to try it run in either jkit or gradle doc project:
```
mvn clean -Pdoc-watch package
```
Make some changes in adoc files and see the site updated when refreshing the browser

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: gradle-plugin/doc/pom.xml [hunk 1]
```diff
@@ -103,26 +112,32 @@
...
-                  <sourceHighlighter>coderay</sourceHighlighter>
-                  <backend>html</backend>
-                  <sourceHighlighter>coderay</sourceHighlighter>
-                  <attributes>
-                    <toc>left</toc>
-                    <linkcss>false</linkcss>
-                    <plugin>${plugin}</plugin>
-                    <task-prefix>${task-prefix}</task-prefix>
-                    <cluster>${cluster}</cluster>
-                    <version>${project.version}</version>
```
Comment (manusa) [inline gradle-plugin/doc/pom.xml:116]: Is this setting not necessary? Is the project.version the default value?

Comment (manusa) [reply gradle-plugin/doc/pom.xml:116]: OK, it's in the parent.

**Review by rohanKanojia — APPROVED** [review]

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**